src/Illuminate/Foundation/Application.php

```diff
@@ -45,7 +45,7 @@ class Application extends Container implements ApplicationContract, CachesConfig
      *
      * @var string
      */
-    const VERSION = '12.23.1';
+    const VERSION = '12.23.2';
```

Comment:
This just to make sure Testbench logic works correctly:

<img width="1221" height="679" alt="image" src="https://github.com/user-attachments/assets/d18a2382-65c3-4d19-9759-dedbc5f6286f" />